tests/h/storage_test.py

```diff
@@ -583,6 +638,15 @@ def scoped_open_group(factories):
     return factories.OpenGroup(scopes=[scope])
 
 
+@pytest.fixture
+def group_service(pyramid_config, factories):
+    open_group = factories.OpenGroup()
+    group_service = mock.Mock(spec_set=['find'])
```

Comment:
I wondered if it would be possible to pass `h.interfaces.IGroupService` directly here to `spec_set` instead of manually enumerating methods. That didn't work in a quick local test though and I haven't looked into why.